docs/data-abstraction/schedule-data-abstraction.md

```diff
@@ -66,6 +67,8 @@
         - :material-notebook-edit-outline: [Calculate Approximate Square Root](https://githubtocolab.com/ProactiveProgrammers/www.proactiveprogrammers.com/blob/master/files/data-abstraction/numerical-computation/calculate-approximate-square-root.ipynb){:target="_blank"}
         - :material-notebook-edit-outline: [Compute an Arithmetic Mean](https://githubtocolab.com/ProactiveProgrammers/www.proactiveprogrammers.com/blob/master/files/data-abstraction/numerical-computation/compute-arithmetic-mean.ipynb){:target="_blank"}
         - :material-notebook-edit-outline: [Perform Integer Primality Testing](https://githubtocolab.com/ProactiveProgrammers/www.proactiveprogrammers.com/blob/master/files/data-abstraction/numerical-computation/perform-primality-testing.ipynb){:target="_blank"}
+- Check out this Engineering Effort on [Primality Testing](./engineering-efforts/primality-testing.md)
```

Comment:
This is a good idea! I like the links to the assignments; this was a feature that students requested during the last iteration of the Data Abstraction Course.